src/library/scala/collection/GenTraversableLike.scala

```diff
@@ -158,20 +158,6 @@ trait GenTraversableLike[+A, +Repr] extends Any with GenTraversableOnce[A] with
   @migration("The behavior of `scanRight` has changed. The previous behavior can be reproduced with scanRight.reverse.", "2.9.0")
   def scanRight[B, That](z: B)(op: (A, B) => B)(implicit bf: CanBuildFrom[Repr, B, That]): That
 
-  /** Applies a function `f` to all elements of this $coll.
```

Comment:
We're trying to simplify the collections, which may mean that the whole Gen hierarchy goes away (since it is generalizing over serial vs. parallel collections).  To prevent extra future work, I wouldn't remove anything from X just because it also is found in GenX.